Terasology/MetalRenegades
# Add trading with NPCs

**To Test**:
 - Go and interact with any NPC
 - After the first page of dialogue, a trading UI should come up
 - Select the item that you want from the left list, select the item that you'll offer from the right list and click the button below
 - If the citizen accepts the trade, you should have the item you selected. Otherwise, you'll get a message saying "Offer declined"

![trade_gif](https://user-images.githubusercontent.com/32035777/62375399-d9120900-b55b-11e9-9c2c-44212b4fe314.gif)
